backend/server.py

```
@@ -5863,12 +5863,49 @@ async def autonomy_status() -> dict[str, object]:
+    # ── Self-bootstrap status: shows whether the platform has onboarded itself
+    #    as a company. Without this, "0 companies" on /api/doctor/public is
+    #    opaque — the operator can't tell if self-bootstrap hasn't run yet,
+    #    failed silently, or is still in progress. ──
+    self_bootstrap_status: dict[str, object] = {"enabled": False}
+    try:
+        from services.self_bootstrap import self_bootstrap_enabled, SELF_WEBSITE_URL, SELF_REPO_URL
+        self_bootstrap_status["enabled"] = self_bootstrap_enabled()
+        self_bootstrap_status["website_url"] = SELF_WEBSITE_URL
+        self_bootstrap_status["repo_url"] = SELF_REPO_URL
+    except Exception:  # pragma: no cover - defensive
+        pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)